components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/APIMgtAdminServiceImpl.java

```diff
@@ -50,15 +54,46 @@ public APIMgtAdminServiceImpl(APISubscriptionDAO apiSubscriptionDAO, PolicyDAO p
     }
 
     /**
-     * @see org.wso2.carbon.apimgt.core.api.APIMgtAdminService#getAPISubscriptions(int)
+     * @see APIMgtAdminService#getPoliciesByLevel(PolicyLevel)
+     */
+    @Override
```

Comment:
I see. will remove.

However, its not only about javadoc. For an example, a developer who is going through this code it will be easy to navigate to the relevant method comment if we have a @see annotation.